src/pages/auth/sign-up/index.page.tsx

```diff
@@ -32,18 +43,24 @@ const SignInPage: NextPageWithLayout = () => {
 
         <BodyForm onSubmit={handleSubmit} paddingTop={22}>
           <CustomText>이름</CustomText>
-          <CustomInput type="text" name="name" required {...name} />
-          <CustomText>아이디</CustomText>
+          <CustomInput name="name" required {...name} />
+          <CustomText>이메일</CustomText>
           <CustomInput
-            type="text"
+            type="email"
             name="id"
             required
             width={pxToRem(204)}
             {...id}
           />
-          <Styled.CheckIdButton onClick={handleIdCheck}>
+          <Styled.CheckIdButton onClick={e => handleIdCheck(e)}>
```

Comment:
```suggestion
          <Styled.CheckIdButton onClick={handleIdCheck}>
```

이렇게 하셔도 됩니다 ㅎㅎ 불필요한 익명함수를 선언하지 않는 습관이 좋아요!